src/screens/ProfileScreen/index.js

```diff
@@ -13,31 +15,20 @@ import {
   TouchableNativeFeedback,
   View
 } from 'react-native';
-import PropTypes from 'prop-types';
-import {SafeAreaProvider} from 'react-native-safe-area-context';
-import {debounce} from 'lodash';
 import {showMessage} from 'react-native-flash-message';
-import {useNavigation} from '@react-navigation/core';
-import netInfo from '@react-native-community/netinfo';
-import AnonymousTab from './elements/AnonymousTab';
+import ImagePicker from 'react-native-image-crop-picker';
+import Toast from 'react-native-simple-toast';
 import ArrowUpWhiteIcon from '../../assets/icons/images/arrow-up-white.svg';
-import BioAndDMSetting from './elements/BioAndDMSetting';
 import BlockComponent from '../../components/BlockComponent';
-import BottomSheetBio from './elements/BottomSheetBio';
-import BottomSheetImage from './elements/BottomSheetImage';
-import BottomSheetRealname from './elements/BottomSheetRealname';
+import {ButtonNewPost} from '../../components/Button';
+import ShadowFloatingButtons from '../../components/Button/ShadowFloatingButtons';
 import CustomPressable from '../../components/CustomPressable';
-import FollowInfoRow from './elements/FollowInfoRow';
-import LinkAndSocialMedia from './elements/LinkAndSocialMedia';
 import PostOptionModal from '../../components/Modal/PostOptionModal';
-import ProfileHeader from './elements/ProfileHeader';
-import ProfilePicture from './elements/ProfilePicture';
-import ProfileTiktokScroll from './elements/ProfileTiktokScroll';
-import RenderItem from '../FeedScreen/RenderList';
-import ShareUtils from '../../utils/share';
-import StorageUtils from '../../utils/storage';
-import dimen from '../../utils/dimen';
-import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
+import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+import {Context} from '../../context';
+import {setFeedByIndex} from '../../context/actions/feeds';
+import {setMyProfileFeed} from '../../context/actions/myProfileFeed';
+import {setMyProfileAction} from '../../context/actions/setMyProfileAction';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been rearranged and some new components are imported. Make sure these components exist and are exported correctly from their respective modules.
```diff
+import ArrowUpWhiteIcon from '../../assets/icons/images/arrow-up-white.svg';
+import BlockComponent from '../../components/BlockComponent';
+import {ButtonNewPost} from '../../components/Button';
+import ShadowFloatingButtons from '../../components/Button/ShadowFloatingButtons';
+import CustomPressable from '../../components/CustomPressable';
+import PostOptionModal from '../../components/Modal/PostOptionModal';
+import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+import {Context} from '../../context';
+import {setFeedByIndex} from '../../context/actions/feeds';
+import {setMyProfileFeed} from '../../context/actions/myProfileFeed';
+import {setMyProfileAction} from '../../context/actions/setMyProfileAction';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->